examples/specs_integration/src/sync/entity_delete.rs

```diff
@@ -0,0 +1,37 @@
+use crate::sync::GdSpatial;
+use godot::*;
+use helper::proc_add_remove_events;
+use specs::prelude::*;
+use specs::world::Index;
+use std::collections::HashMap;
+
+pub struct GDDeleteOldSpatials {
+    gdspatial_reader: ReaderId<ComponentEvent>,
+}

```

Comment:
Unlike `GdEntityCreation` and `GdEntityTransform` this is not a Specs `System`. Is there anything preventing it from being one? Maybe if could be documented?